README.adoc

```diff
@@ -2548,19 +2548,19 @@ some_method('w', 'x', 'y') # => 'y, 2, w, x'
 some_method('w', 'x', 'y', 'z') # => 'y, z, w, x'
 ----
 
-=== Keyword Arguments Order [[keyword-arguments-order]]
+=== Keyword Arguments Order
```

Comment:
Please keep the anchor.

Reply:
Ooops, just checked Bozhidar's comment on anchor redundancy. Please disregard my previous comment.

Reply:
@pirj but @bbatsov asked me to remove this anchor one comment above...

> The explicit anchor here is redundant. There are a bunch of anchors in the guide for historical reasons (the migration from the original markdown format).
https://github.com/rubocop/ruby-style-guide/pull/771#discussion_r628362808